pkg/helper/pipelinerun/description/description.go

```diff
@@ -24,6 +24,11 @@ const templ = `{{decorate "bold" "Name"}}:	{{ .PipelineRun.Name }}
 {{decorate "bold" "Service Account"}}:	{{ .PipelineRun.Spec.ServiceAccountName }}
 {{- end }}
 
+{{- $timeout := getTimeout .PipelineRun -}}
+{{- if ne $timeout "" }}
+{{decorate "bold" "Timeout\n"}} {{ .PipelineRun.Spec.Timeout.Duration.String }}
```

Comment:
We also want it to follow the pattern of what is in the `.golden` files for output:

```
Name:        tr-1
Namespace:   ns
Task Ref:    t1
Timeout:     1h0m0s
```

So

```
{{decorate "bold" "Timeout"}}: {{ .PipelineRun.Spec.Timeout.Duration.String }}
```

Reply:
Done